.github/workflows/release.yml

```diff
@@ -0,0 +1,35 @@
+name: Release Helm Charts
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    env:
+      HELM_VERSION: v3.5.2
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Fetch history
+        run: git fetch --prune --unshallow
```

Comment:
Is it needed to fetch the history? The [chart-releaser-action doc](https://github.com/helm/chart-releaser-action#example-workflow) does like below.

```yaml
      - name: Checkout
        uses: actions/checkout@v2
        with:
          fetch-depth: 0
```

Reply:
`fetch-depth: 0` is also OK, This seems to be useful for large repositories.

https://github.com/actions/checkout#fetch-all-history-for-all-tags-and-branches

ref:
https://github.com/prometheus-community/helm-charts/blob/main/.github/workflows/release.yaml
https://stackoverflow.com/questions/3954852/how-to-complete-a-git-clone-for-a-big-project-on-an-unstable-connection

Reply:
Thanks. I realized that `fetch-depth: 0` is the same as `git fetch --unshallow`. I believe both methods work the same, but I think the official `helm/chart-releaser-action` way is simpler, don't you?

Reply:
I don't think it's a good idea to set the `fetch-depth` to `0`.
As the history increases in the future, it may fail to checkout the source, So I think the current method is fine.

Reply:
So do we still need `git fetch --prune --unshallow`?

Reply:
Yes, need 2 steps. 
- Checkout with [fetch-depth : 1 (default)](https://github.com/actions/checkout/blob/61b9e3751b92087fd0b06925ba6dd6314e06f089/src/input-helper.ts#L85)
- Fetch history with `--unshallow`

Reply:
@ymorimo 

Thanks for the discussion. Fixed with `fetch-depth: 0` in e6e6b9b.